classes/class04/homework/routes/ingredients.js

```diff
@@ -0,0 +1,50 @@
+var models = require('./models.js');
+
+module.exports.getList = function(req,res){
+	//Gets list of ingredients from DB
+	models.Ingredients.find({}).exec(function(err,data){
+		if(err){res.status(500).send("Failed to access list.")}
+		res.render('ingredientsList',{layout:'empty', 'list':data});
+	});
+};
+
+module.exports.addNew = function(req,res){
+	if(req.body.name == "clear"){ //Debugging codeword for clearing ingredients
```

Comment:
Not generally good practice. If you need to delete something, have a separate route.